tests/integration/cmd_app_test.go

```diff
@@ -0,0 +1,62 @@
+package integration
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/openshift/odo/tests/helper"
+	"os"
+	"path/filepath"
+	"time"
+)
+
+var _ = Describe("odoCmdAppE2e", func() {
+	var project string
+	var context string
+
+	appName := "app"
+	cmpName := "nodejs"
+
+	// This is run after every Spec (It)
+	var _ = BeforeEach(func() {
+		SetDefaultEventuallyTimeout(10 * time.Minute)
+		project = helper.CreateRandProject()
+		context = helper.CreateNewContext()
+	})
+
+	// Clean up after the test
+	// This is run after every Spec (It)
+	var _ = AfterEach(func() {
+		helper.DeleteProject(project)
+		os.RemoveAll(".odo")
```

Comment:
Ahh, i did not notice it in my first review. 

Please remove this as no component is created in the integration test dir.